package.json

```diff
@@ -75,6 +89,24 @@
           "type": "string",
           "default": "",
           "description": "Path to the virtual environment activation script. Use only if you have a custom activation script. It will be sourced using bash before executing Ansible commands. If set, the Interpreter Path setting is ignored."
+        },
+        "ansible.navigate.executablePath": {
```

Comment:
`ansible.ansibleLint.path` is used for `ansible-lint` executable path and `ansible.ansible.path` for ansible excutable.
I think the naming convention should follow a uniform pattern. How about `ansible.ansibleNavigator.path` and `ansible.ansiblePlaybook.path`?